draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1121,7 +1134,7 @@ uCDN-A                         dCDN-B                         dCDN-C
                 <t>
                 dCDN-C acts upon the trigger. However, the command fails at dCDN-C as,
                 for example, the trigger definition contains an "action" type that is not supported by dCDN-C.
-                dCDN-C's action is depicted by 7.2 in the diagram, while 7.1 shows dCDN-B acting on its own trigger.
+                dCDN-C's action is depicted by 7.2 in the diagram, while 7.1 shows dCDN-B acting on its trigger.
```

Comment:
I think removing "own" just made "its" more ambiguous.  This could be read in such a way that dCDN-B is acting on the trigger of dCDN-C.  I recommend reverting this change.